src/var.c

```diff
@@ -54,8 +61,15 @@ struct fi_param_entry {
 	struct dlist_entry entry;
 };
 
+struct fi_conf_entry {
```

Comment:
use ofi prefix, no need to make this worse

Reply:
done, I was just being consistent with what was there